visualization/include/pcl/visualization/impl/pcl_visualizer.hpp

```diff
@@ -1702,11 +1702,11 @@ pcl::visualization::PCLVisualizer::addPolygonMesh (
     // Create polys from polyMesh.polygons
     vtkSmartPointer<vtkCellArray> cell_array = vtkSmartPointer<vtkCellArray>::New ();
     
-    int idx = details::fillCells(lookup,vertices,cell_array);
+    const auto idx = details::fillCells(lookup,vertices,cell_array);
```

Comment:
If the cast is needed, let's have fillCells return vtkIdType itself

Reply:
Its unsigned long long to long long - I just thought it is, but I'm not 100% sure?

Reply:
Putting the case in fillCells (and the correct return type) would be safer, despite whatever the type be